src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -14,6 +14,7 @@
 using System.Management.Automation.Internal;
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-macos`
method conversion.Correctly casts 100n to value 100 as type [int]
```powershell
Expected '100', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 510
510:         $Result | Should -Be $Value
```